Sources/MockoloFramework/Operations/Generator.swift

```diff
@@ -107,12 +107,21 @@ public func generate(sourceDirs: [String],
     let t2 = CFAbsoluteTimeGetCurrent()
     log("Took", t2-t1, level: .verbose)
     
-    let typeKeyList = [parentMocks.compactMap {$0.key.components(separatedBy: "Mock").first}, annotatedProtocolMap.map {$0.key}].flatMap{$0}
-    var typeKeys = [String: String]()
-    typeKeyList.forEach { (t: String) in
-        typeKeys[t] = "\(t)Mock()"
-    }
-    SwiftType.customTypeMap = typeKeys
+    let typeKeyList = [
+        parentMocks.compactMap { (key, value) -> String? in
+            if value.entityNode.mayHaveGlobalActor {
+                return nil
+            }
+            return key.components(separatedBy: "Mock").first
+        },
+        annotatedProtocolMap.filter { !$0.value.entityNode.mayHaveGlobalActor }.map(\.key)
+    ]
+        .flatMap { $0 }
+        .map { typeName in
+            // nameOverride does not work correctly but it giving up.
```

Comment:
[nits]

I am sorry, but could you add a meaningful prefix such as FIXME so that we can easily address its problem in the future. What do you think of it?

Reply:
I am not sure I want this fixed.
If you think anything in this comment, it might be better there are no comments.

Reply:
I got it. I just concerned if nameOverride support is hard of not. Thank you for your confirmation.